repository_service_tuf_api/api/metadata.py

```diff
@@ -19,7 +19,8 @@
 @router.post(
     "/",
     summary=(
-        f"Rotate role metadata. Scope: {SCOPES_NAMES.write_metadata.value}"
+        f"Rotate role metadata. Scope: "
+        f"{SCOPES_NAMES.write_metadata_sign.value}"
```

Comment:
This endpoint is the metadata update.
```suggestion
        f"{SCOPES_NAMES.write_metadata.value}"
```